OpenRA.Game/Server/Server.cs

```diff
@@ -53,6 +53,7 @@ public class Server
 		public ServerSettings Settings;
 		public ModData ModData;
 		public Map Map;
```

Comment:
It would be worth adding an extra newline before this, and a comment `// Managed by LobbyCommands`.